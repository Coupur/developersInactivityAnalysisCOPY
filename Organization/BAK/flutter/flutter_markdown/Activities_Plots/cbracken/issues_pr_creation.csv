id,date,creator_login
309543909,2018-03-28 21:46:59,cbracken
