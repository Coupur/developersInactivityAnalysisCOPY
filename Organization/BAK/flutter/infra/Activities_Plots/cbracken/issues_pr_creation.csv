id,date,creator_login
413646617,2019-02-23 02:45:36,cbracken
