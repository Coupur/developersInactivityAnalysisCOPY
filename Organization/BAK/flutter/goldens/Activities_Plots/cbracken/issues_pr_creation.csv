id,date,creator_login
339956461,2018-07-10 18:24:10,cbracken
339984604,2018-07-10 19:55:20,cbracken
340357731,2018-07-11 18:35:02,cbracken
